rolling/ci-drake-ros-underlay.yaml

```diff
@@ -0,0 +1,64 @@
+%YAML 1.1
+# ROS buildfarm ci-build file
+---
+build_environment_variables:
+  ROS_PYTHON_VERSION: '3'
+build_tool: colcon
+build_tool_args: '--merge-install --cmake-args -DCMAKE_BUILD_TYPE=Release -DSKIP_MULTI_RMW_TESTS=1 --no-warn-unused-cli'
+build_tool_test_args: '--merge-install --retest-until-pass 2 --ctest-args -LE "(linter|performance|xfail| placeholder-with-space )" --pytest-args -m "not linter and not performance and not xfail"'
```

Comment:
The `placeholder-with-space` is there to keep the ros_buildfarm shell pipeline from dropping the quotes and thus causing parser problems with the regex metacharaters right? Do you think that merits a comment or just a passing sigh of exasperation?

Reply:
You're correct. We should probably open a ticket on `ros_buildfarm`, though the behavior isn't consistent across all platforms on ci.ros2.org either.

A sigh of exasperation is necessary regardless.